dbms/src/Storages/Transaction/ProxyFFIStatusService.cpp

```diff
@@ -31,11 +33,26 @@ HttpRequestRes HandleHttpRequestSyncStatus(
 {
     HttpRequestStatus status = HttpRequestStatus::Ok;
     TableID table_id = 0;
+    pingcap::pd::KeyspaceID keyspace_id = NullspaceID;
     {
-        std::string table_id_str(path.substr(api_name.size()));
+        auto * log = &Poco::Logger::get("HandleHttpRequestSyncStatus");
+        LOG_DEBUG(log, "handling sync status request, path: {}, api_name: {}", path, api_name);
+
+        // Query schema: /keyspace/{keyspace_id}/table/{table_id}
```

Comment:
do we keep the old query schema without keyspace?

Reply:
Yes, the TiFlash replica status might be uncheckable in the rolling upgrade, we support the old schema.